docs/src/index.md

```diff
@@ -1,21 +1,46 @@
 # Welcome to the `NoteMate.jl` Docs! 👋
 
-> The perfect companion for working with your notes 
+> The perfect companion for working with your notes 📚
 
-NoteMate is a programmatic transformation tool used to parse and manipulate files whose heading structure and contents follow [the Open Knowledge Model](https://jacobzelko.com/04172022033744-open-knowledge-standard/index.html). Due to the way NoteMate internally represents documents, it is possible to transform notes from different structures to the Open Knowledge Model standard, and to express the standard in different output formats.
+`NoteMate.jl` is a programmatic transformation tool used to parse and manipulate files whose structure follows the [Open Knowledge Model (OKM)](https://jacobzelko.com/04172022033744-open-knowledge-standard/index.html).
+The goal of `NoteMate.jl` is to enable one to focus only on writing without worry of needing to add any special markup to your note *while* having the benefits of quickly sharing thoughts.
+Due to the way `NoteMate.jl` parses documents, `NoteMate.jl` can transform notes following the OKM and express them in different outputs irrespective of the implementation.
 
-NoteMate helps build [digital gardens](https://www.lekoarts.de/garden/what-is-a-digital-garden) for the sharing of ideas, insights and information synthesis. It works with files that empathize native human readability and manipulation without dependency on complex machine-readable metadata linkages in the frontend. Citations are inspired by academic standards and can be natively copied and pasted from the notes with no further interpretation needed. This makes it a good choice for building digital gardens that can be grown with a high-quality, low-tech evidence chain. 
```

Comment:
Do we want to strike this out entirely? I would at least leave the premise of "hey, you can write some pretty neat digital gardens with this that excell at evidence chaining" in here to offer people a cookie, so to speak. Otherwise this feels a bit too dry for "and why should I care about this" besides the long-term vision we offer later.

Reply:
I agree. Done.